### ISSUE TITLE
Classical Loss Functions in Machine Learning | Shiyu Chen

### ISSUE THREAD

**Issue by chenshiyuhit** [chenshiyuhit/homepage-comments#29]
https://www.chenshiyu.top/blog/2019/08/05/Classical-Loss-Functions-in-Machine-Learning/ 

 Some classical loss functions in ML, eg. Cross-Entropy.
